src/rgw/setup/templates/rgw.logrotate.tmpl

```diff
@@ -1,5 +1,7 @@
-TMP_LOG_PATH/*.log
-{
+TMP_LOG_PATH/radosgw-admin.log
+TMP_LOG_PATH/rgw_setup.log
+TMP_LOG_PATH/rgw_support_bundle.log
+TMP_LOG_PATH/rgw-??.log{
```

Comment:
what rgw-??.log will do, its a wildcard to pickup any file with .log extension ? It wont create same problem as reported in defect ?

Reply:
rgw-??.log only allows 0-2 char  ie rgw-1.log - rgw-99.log rgw-aa.log 
but when we rotate the log, it adds timestamp as rgw-1.log-2022-03-31-1648710061.log so this will be filtered out 

**also only `*`ie any number or char  and `?` ie 0 or 1 char is supported in logrotate syntax.

Reply:
Thanks, understood more in detail after our discussion, am fine